frontend/src/metabase/entities/databases/big-query-fields.js

```diff
@@ -69,56 +69,59 @@ export default function getFieldsForBigQuery(details) {
         : [
             {
               name: "project-id",
-              "display-name": "Project ID",
-              placeholder: "praxis-beacon-120871",
+              "display-name": t`Project ID`,
+              "helper-text": t`Project ID to be used for authentication. You can omit this field if you are only querying datasets owned by your organization.`,
+              placeholder: "1w08oDRKPrOqBt06yxY8uiCz2sSvOp3u",
               required: true,
             },
           ]),
       {
         name: "dataset-id",
-        "display-name": "Dataset ID",
-        placeholder: "toucanSightings",
+        "display-name": t`Dataset ID`,
+        "helper-text": t`Make sure to leave out the Project ID prefix in "project_name:dataset_id" and only enter “dataset_id”`,
```

Comment:
Should we change this for the new BigQuery driver as well?

Reply:
Thanks! I've updated copy for the new driver. It's on the backend.